langfuse/serializer.py

```diff
@@ -1,13 +1,16 @@
+from collections.abc import Sequence
+from dataclasses import asdict, is_dataclass
 from datetime import date, datetime
-from dataclasses import is_dataclass, asdict
 from json import JSONEncoder
 from typing import Any
 from uuid import UUID
 
-from langfuse.api.core import serialize_datetime
-
+from google.protobuf.json_format import MessageToJson
+from google.protobuf.message import Message
 from pydantic import BaseModel
```

Comment:
the re-ordering is done by [`Black`](https://pypi.org/project/black/) formatter, and since you don't have a CONTRIBUTING guide into this repo, so not sure which standard you follow.